src/css/style/fixedDataTableCell.css

```diff
@@ -29,10 +29,9 @@
   background-color: #0284ff;
 }
 .public/fixedDataTableCell/hasReorderHandle .public/fixedDataTableCell/cellContent {
-  margin-left: 12px;
+  padding-left: 20px;
```

Comment:
Margin was added here so as to leave some space if the reorder handle exists. 
But now, the only wrapper element has display set to `table-cell`, and such elements are unable to have margin. 
So we'll just add these to padding instead. It should have the same effect for most use cases.

But if the user wants to add margin, he can either 
* change the display property
* change padding
* just make a new wrapper element